plugin-cgroupv1/src/oar2/probe.rs

```diff
@@ -1,26 +1,39 @@
 use alumet::{
-    measurement::{MeasurementAccumulator, Timestamp},
-    metrics::TypedMetricId,
+    measurement::{MeasurementAccumulator, MeasurementPoint, Timestamp},
     pipeline::{elements::error::PollError, Source},
-    plugin::util::CounterDiff,
 };
-use std::result::Result::Ok;
+use std::{collections::HashMap, result::Result::Ok};
 
-use super::utils::OpenedCgroupv1;
-
-use crate::cgroupv1::gather_value;
+use crate::cgroupv1::{Cgroupv1Probe, Metrics};
 
 pub struct OAR2Probe {
-    pub cpu_metric_counter_diff: CounterDiff,
-    pub cpu_metric: TypedMetricId<u64>,
-    pub memory_metric: TypedMetricId<u64>,
-    pub oar2_metric_file: OpenedCgroupv1,
+    cgroupv1: Cgroupv1Probe,
+    additional_attrs: HashMap<String, String>,
```

Comment:
So the jobID is lost between additional attributs ? 😲
We can discuss about it

Reply:
yes cause cgroupv1 actually don't have a concept of "job". So even if submodules OAR2 and Slurm both have this concept, I think it doesn't make sense to add this semantic in cgroupv1 related code.

Reply:
We already are in the OAR2 submodules,  so I don't wanted to add the jobID in the crgoupv1 struct, I was more in  a way to add an other field in the struct of the oar2 which contain a jobID and an other one containing all other attributes such as:
```suggestion
    cgroupv1: Cgroupv1Probe,
    jobID: u64
    additional_attrs: HashMap<String, String>,
```

Reply:
ah yes sorry! :P 
hum in fact there's no need to store jobID as a field here since it's registered in the additional_attrs at the creation of the OAR2Probe.